### DISCUSSION TITLE
Observational data for Antarctic baseline

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#40] [Antarctica]
I recently shared a scheduling poll and email with everyone on the [Team Member](https://docs.google.com/spreadsheets/d/18DBHRbEAuquOqChHN2GqUR2uPKpW7D1Y-IBqMPM6TtM/edit#gid=1372946030) spreadsheet who has selected `Remote Sensing` AND (`Antarctica` OR `Glacier`).

Based on responses we've scheduled two meetings (times listed below). I won't post the Zoom link publicly, but if anyone else wants to join, let me know.

If you can't join one of the meetings, this thread can also be used to discuss the issue prior to, between, and after the Zoom meetings. I'll post the meeting recordings here too.

Goal: Select appropriate existing data sets that define ice sheet vs. peripheral glacier in Antarctica. If these products do not already exist, or exist but do not meet our needs, then select upstream remote sensing product, methods, and rule sets to define ice sheet vs. peripheral glaciers for Antarctica. This is harder than in Greenland due to ice shelves, grounding lines, more sea ice, and less visible ground. Suggestions are welcome...

Meeting 1
```
May 20 09:00 -0800 AKDT US/Alaska
May 20 10:00 -0700 PDT US/Pacific
May 20 13:00 -0400 EDT US/Eastern
May 20 17:00 +0000 UTC UTC
May 20 18:00 +0100 BST Europe/London
May 20 19:00 +0200 CEST Europe/Brussels
```

Meeting 2
```
May 24 02:15 -0700 PDT US/Pacific
May 24 05:15 -0400 EDT US/Eastern
May 24 09:15 +0000 UTC UTC
May 24 10:15 +0100 BST Europe/London
May 24 11:15 +0200 CEST Europe/Brussels
```

**Comment by oien2** [comment]
Hi Ken,

Can I get a link to the zoom meeting for Friday?

-Rachel

On Mon, 6 May 2024 at 21:26, Ken Mankoff ***@***.***> wrote:

> I recently shared a scheduling poll and email with everyone on the Team
> Member
> <https://docs.google.com/spreadsheets/d/18DBHRbEAuquOqChHN2GqUR2uPKpW7D1Y-IBqMPM6TtM/edit#gid=1372946030>
> spreadsheet who has selected Remote Sensing AND (Antarctica OR Glacier).
>
> Based on responses we've scheduled two meetings (times listed below). I
> won't post the Zoom link publicly, but if anyone else wants to join, let me
> know.
>
> If you can't join one of the meetings, this thread can also be used to
> discuss the issue prior to, between, and after the Zoom meetings. I'll post
> the meeting recordings here too.
>
> Goal: Select appropriate existing data sets that define ice sheet vs.
> peripheral glacier in Antarctica. If these products do not already exist,
> or exist but do not meet our needs, then select upstream remote sensing
> product, methods, and rule sets to define ice sheet vs. peripheral glaciers
> for Antarctica. This is harder than in Greenland due to ice shelves,
> grounding lines, more sea ice, and less visible ground. Suggestions are
> welcome...
>
> Meeting 1
>
> May 20 09:00 -0800 AKDT US/Alaska
> May 20 10:00 -0700 PDT US/Pacific
> May 20 13:00 -0400 EDT US/Eastern
> May 20 17:00 +0000 UTC UTC
> May 20 18:00 +0100 BST Europe/London
> May 20 19:00 +0200 CEST Europe/Brussels
>
> Meeting 2
>
> May 24 02:45 -0700 PDT US/Pacific
> May 24 05:45 -0400 EDT US/Eastern
> May 24 09:45 +0000 UTC UTC
> May 24 10:45 +0100 BST Europe/London
> May 24 11:45 +0200 CEST Europe/Brussels
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/IACS-cryo/Delineation-WG/discussions/40>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADGRBX267ZOUZC4GEGIAHUTZA7RPZAVCNFSM6AAAAABHJWKNGKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGYYTSNZZGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Dr Rachel P. Oien
Teaching Fellow, Dept. of Geography, Durham University
PhD in Geoscience, University of Aberdeen, U.K. '21
***@***.***
+44 7548 751666

**Comment by mankoff** [comment]
Hello,

We had a discussion about Antarctic baseline products. The video conversion failed so only the first half of the meeting is capture. It also converted the individual speakers full-screen, not the overview panel, so it's a bit jumpy. Audio may be complete - haven't checked. I've uploaded whatever Zoom generated.

https://drive.google.com/drive/folders/14zSI5y8wsH4vsdTJB1ohRsw96-vzNo6-?usp=drive_link

The Friday meeting, same topic but different times is at

May 24 02:45 -0700 PDT US/Pacific
May 24 05:45 -0400 EDT US/Eastern
May 24 09:45 +0000 UTC UTC
May 24 10:45 +0100 BST Europe/London
May 24 11:45 +0200 CEST Europe/Brussels

I'm not going to post a Zoom link publicly, and Fabien is hosting that meeting. Please email him or me if you would like to join and do not already have a calendar invite.

After the Friday meeting we'll summarize the results here for further discussion.

**Comment by mankoff** [comment]
I've ingested some products (7 Greenland, only 3 for Antarctica: RGI, BedMachine and ESA CCI Landcover) and started generating some summary statistics, graphics, and a plan-view map of overlap.

(Note that this plan view display has already been done in Antarctica by Hansen (2022; Fig. 1) http://doi.org/10.5194/tc-16-711-2022 although there only focusing on SMB RCMs)

Code and graphics are at https://github.com/IACS-cryo/Delineation-WG/blob/main/D1/set_graphic.org (that is, a digital workbook in the D1 folder as this overview is for Deliverable 1).

@willkochtitzky and others who want to work with data - It's easy to share raw data (go to the source) or final data (although preparing a well-formatted NetCDF takes time). I'm not sure of the best way to share a bunch of semi-processed data. I've generated 30 masks (breaking products into all, main, peripheral, shelf, etc.). I've dumped all the masks I've generated into a ZIP file in our project Google Drive under Greenland/masks_GL.zip and Antarctica/masks_AQ.zip. I expect these files are temporary and may be removed at some point.
https://drive.google.com/drive/folders/1wD3TpKi4tGnDVGIhtnPaKjg6Wxv_Rj0X?usp=drive_link

**Reply by willkochtitzky** [reply]
Thanks Ken! I am heading out of town tonight for two weeks, but will try to look at this more in mid-June. Thank you!
